modules/local/openms_mapaligneridentification.nf

```diff
@@ -1,14 +1,14 @@
 process OPENMS_MAPALIGNERIDENTIFICATION {
-    tag "$meta.id"
+    tag "$merge_id.id"
```

Comment:
The first item should always go by `meta`

Reply:
I now store all metadata corresponding to a merge event in a separate map [group_meta, [meta1,meta2,meta3]] and join them to processes if we need to split them again